packages/orchestration/README.md

```diff
@@ -38,6 +41,16 @@ The orchestration service provides essential features like templating and conten
 
 Find more details about orchestration workflow [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow).
 
+## Relationship between Orchestration and Resource Groups
+
+SAP AI Core manages access to orchestration of generative AI models through the global AI scenario `orchestration`.
+Creating a deployment for enabling orchestration capabilities requires access to this scenario.
+
+[Resource groups](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/resource-groups?q=resource+group) represent a virtual collection of related resources within the scope of one SAP AI Core tenant.
+Each resource group allows for a one-time orchestration deployment.
+
+Consequently, each orchestration deployment uniquely maps to a resource group within the `orchestration` scenario.
```

Comment:
As, the deployment requirements change between orchestration and foundation-models. I would prefer to write about it in the dedicated client document  instead of a separate file, as this would be more confusing.